src/Admin/Admin.php

```diff
@@ -261,6 +263,22 @@ protected function enableReports(): bool {
 		return apply_filters( 'woocommerce_gla_enable_reports', true );
 	}
 
+	/**
+	 * Add suggested privacy policy content
+	 *
+	 * @return void
+	 */
+	public function privacy_policy() {
+		$content = '<p class="privacy-policy-tutorial">' . sprintf(
+			/* translators: 1) HTML anchor open tag 2) HTML anchor closing tag */
+			esc_html__( 'By using this extension, you may be storing personal data or sharing data with an external service. %1$sLearn more about what data is collected by Google and what you may want to include in your privacy policy.%2$s.', 'google-listings-and-ads' ),
```

Comment:
There's an extra full stop at the end of the paragraph.

<img width="793" alt="Screenshot 2023-09-21 at 15 28 53" src="https://github.com/woocommerce/google-listings-and-ads/assets/914406/1384ad87-1b67-4414-9ad6-c61355075926">